spec/server.spec.js

```diff
@@ -0,0 +1,19 @@
+//const app = require('../server');
+var request = require('request');
+
+describe('server', function() {
+    it('should respond to GET requests for /reviews with a 200 status code', function (done) {
+        request('http://127.0.0.1:3000/reviews', function(error, response, body) {
+            expect(response.statusCode).toBe(200);
+            done();
+          });
+    });
+
+    it('Should 404 when asked for a nonexistent endpoint', function(done) {
+        request('http://127.0.0.1:3000/arglebargle', function(error, response, body) {
```

Comment:
I think it could be good if we also test that our server responds with 404 if it gets a room id that it doesn't have (ie ID # 100000).  I think that would mean you'll want to have a check for that in your db/server.

Reply:
sure i will include